app/containers/RoomItem/Actions.tsx

```diff
@@ -21,6 +21,8 @@ import I18n from '../../i18n';
 const CONDENSED_ICON_SIZE = 24;
 const EXPANDED_ICON_SIZE = 28;
 
+const starColor = '#f5d100';
```

Comment:
This feels wrong.
There might be no reason to make it part of any theme, but shouldn't we unify colors?
I wonder how web deals with the fav icon.

Reply:
It has no color